Data/README.md

```diff
@@ -19,9 +21,11 @@ These programs also work within [SAS University Edition](https://www.sas.com/en_
   You can see examples of the code and some sample visualizations in [the NYT-COVID19-SAS notebook](NYT-COVID19-SAS.ipynb). (This notebook can be added to SAS University Edition, which supports Jupyter Lab.  You can also enable the [SAS kernel for your own SAS environment](https://blogs.sas.com/content/sasdummy/2016/04/24/how-to-run-sas-programs-in-jupyter-notebook/).)
 * [import-data-jhu.sas](./import-data-jhu.sas) - This program uses PROC HTTP to fetch the daily CSV files from [https://github.com/CSSEGISandData/COVID-19](https://github.com/CSSEGISandData/COVID-19). It uses PROC IMPORT to read the data a single SAS data set, covering all countries/regions over all of the days with data. Subsequent steps are used to clean/normalize the data for easier reporting. The main outputs include: a data set with all countries and regions/states and daily data with running totals of cases/deaths, and a data set summarized to the country level with "days to double" rate for cases/deaths.
 
+*	[import-data-ihme.sas](./import-data-ihme.sas) - This program uses PROC HTTP to fetch the CSV files from http://www.healthdata.org/covid/datadownloads. Subsequent steps are used to clean/normalize the data for easier reporting. The main output is a data set with all archived projection estimates of the demand for hospital services, including the availability of ventilators, general hospital beds, and ICU beds, as well as daily and cumulative deaths due to COVID-19.
```

Comment:
Can you move this bullet after the next paragraph about the JHU notebook?  That one goes with the previous bullet.